server.js

```diff
@@ -8,7 +8,7 @@ var port = process.env.PORT || 8080;
 
 
 app.use('/bower_components', express.static(__dirname + '/bower_components'));
-app.use(express.static(__dirname + '/public/dist'));
+app.use(express.static(__dirname + '/public'));
```

Comment:
Probably want to change this before pushing up to heroku.  A better option might be to store the static route in a variable depending on what the environment is ... maybe something like:

```
var staticPath = '/public';

if (process.env.ENVIRONMENT && process.env.ENVIRONMENT == 'production') {
    staticPath = '/public/dist';
}

app.us(express.static(__dirname + staticPath));
```

That way, if there is an ENVIRONMENT variable and it's set to production, it'll use the production directory. Otherwise, it'll use public.  So all you'd have to do is add a new variable to heroku called ENVIRONMENT and give it a value of production.